moviepy/audio/io/readers.py

```diff
@@ -110,7 +110,12 @@ def read_chunk(self,chunksize):
         # chunksize is not being autoconverted from float to int
         chunksize = int(round(chunksize))
         L = self.nchannels*chunksize*self.nbytes
-        s = self.proc.stdout.read(L)
+        out, err = self.proc.communicate(timeout=15)
+        if err == b'':
```

Comment:
What about py2?

Reply:
py2 works for that problematic creative url so didn't know if we should change anything there...